app/code/community/Shiphawk/Order/Model/Command/SendOrder.php

```diff
@@ -71,12 +71,17 @@ public function execute(Mage_Sales_Model_Order $order)
               $source_system_id = $item->getItemId();
             }
 
+            if ($item->getProductType() == 'simple') {
+                $price = $item->getProduct()->getFinalPrice();
```

Comment:
is there any chance that `$item->getProduct()`  empty or null?

Reply:
@Icmus23 On line 49 we load the product from the catalog. Can it be used for getting the final price?

Reply:
@vasiliysablin It is possible, I guess, let me check it